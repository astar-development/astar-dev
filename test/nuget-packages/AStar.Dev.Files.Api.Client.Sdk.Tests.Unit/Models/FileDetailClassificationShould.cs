using AStar.Dev.Files.Api.Client.SDK.Models;
using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Files.Api.Client.Sdk.Models;

[TestSubject(typeof(FileDetailClassification))]
public class FileDetailClassificationShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => new FileDetailClassification { ClassificationId = 123, FileDetail = new (), FileDetailClassifications = new (), FileId = 312 }
           .ToJson()
           .ShouldMatchApproved();
}
