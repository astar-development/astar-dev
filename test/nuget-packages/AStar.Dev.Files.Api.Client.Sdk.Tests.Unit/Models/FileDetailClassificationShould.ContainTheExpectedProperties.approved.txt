{
  "fileId": 312,
  "fileDetail": {
    "id": 0,
    "fileAccessDetail": {
      "id": 0,
      "detailsLastUpdated": null,
      "lastViewed": null,
      "softDeleted": false,
      "softDeletePending": false,
      "moveRequired": false,
      "hardDeletePending": false
    },
    "fileName": "",
    "directoryName": "",
    "fullNameWithPath": "\\",
    "height": 0,
    "width": 0,
    "fileSize": 0,
    "isImage": false
  },
  "classificationId": 123,
  "fileDetailClassifications": {
    "classificationId": 0,
    "name": ""
  }
}